src/test/ui/imports/unused-imports-in-test-module.stderr

```diff
@@ -0,0 +1,170 @@
+error: unused import: `std::io::BufRead`
+  --> $DIR/unused-imports-in-test-module.rs:3:5
+   |
+LL | use std::io::BufRead;
+   |     ^^^^^^^^^^^^^^^^
+   |
+note: the lint level is defined here
+  --> $DIR/unused-imports-in-test-module.rs:1:9
+   |
+LL | #![deny(unused_imports)]
+   |         ^^^^^^^^^^^^^^
+
+error: unused import: `super::a`
+  --> $DIR/unused-imports-in-test-module.rs:9:9
+   |
+LL |     use super::a;
+   |         ^^^^^^^^
+   |
+help: consider adding a `#[cfg(test)]` to the parent module
+  --> $DIR/unused-imports-in-test-module.rs:8:1
+   |
+LL | mod test {
+   | ^^^^^^^^
```

Comment:
Seems like it'd be good to add tests for out-of-line modules too (i.e,, `mod test;`). But I'm not sure how feasible it is to test that.